src/screens/BackupSheet.js

```diff
@@ -230,6 +230,11 @@ export default function BackupSheet() {
     additionalTopPadding = true;
   }
 
+  if (android && step === WalletBackupStepTypes.existing_user) {
+    // on this step we have 3 lines of text instead of 2
```

Comment:
I think it might be screen size dependent? On a phone with a bit more weird screen size ratio it might reduce to 2 or expand to 4 lines maybe?

Reply:
Can we do it using the navigation bar height?

Reply:
The whole thing is calculated by some hardcoded values. I just added one more :)